package.json

```
@@ -57,6 +57,10 @@
       "no-extra-semi": 1
     }
   },
+  "engines" : {
+    "node" : ">=10",
+    "npm": ">=5"
+  },
```

```suggestion
  "engines" : {
    "node" : ">=12",
    "npm": ">=6"
  },
```

Currently in [node.js homepage](https://nodejs.org/en/), it is showing v12 in the LTS, and [npm](https://www.npmjs.com/package/npm/v/6.14.6) v6 has released more than an year. So they should be pretty mature.